src/components/ValidateConfigs.ts

```diff
@@ -0,0 +1,119 @@
+import { Component, createElement } from "react";
+
+import { DropdownFilterContainerProps, ListView } from "../components/DropdownFilterContainer";
+import { Alert } from "../components/Alert";
+
+export interface ValidateConfigProps extends DropdownFilterContainerProps {
+    inWebModeler?: boolean;
+    filterNode?: HTMLElement;
+    targetGrid?: ListView;
```

Comment:
Rename everywhere in this widget something with `Grid`. Either more generic like `targetWidget` or more specific `targetListView`

Please check with Guy, to make it consistent in the widgets.

Reply:
Done